plaso/parsers/sqlite_plugins/kik_ios.py

```diff
@@ -45,7 +45,7 @@ class KikIOSPlugin(interface.SQLitePlugin):
   kik.sqlite
   """
 
-  NAME = 'kik_messenger'
```

Comment:
I opt to add a preset `kik_messenger` that points to `kik_ios` for backwards compatibility, similar to `mactime`

Reply:
This is a plugin though, rather than a parser, so I'd have to name the preset "sqlite/kik_messenger" which I don't think is optimal - WDYT?

Reply:
good point, don't that is supported atm.